### DISCUSSION TITLE
openapi/docs get_swagger_ui_html not uses swagger_js_url/swagger_css_url/swagger_favicon_url

### DISCUSSION THREAD

**Discussion by Moodyqiqi** [fastapi/fastapi#13039] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
-- applications.py

1020                return get_swagger_ui_html(
1021                    openapi_url=openapi_url,
1022                    title=f"{self.title} - Swagger UI",
1023                    oauth2_redirect_url=oauth2_redirect_url,
1024                    init_oauth=self.swagger_ui_init_oauth,
1025                    swagger_ui_parameters=self.swagger_ui_parameters,
1026                )
```

### Description

Hi, 

in openapi/docs get_swagger_ui_html, allowed to pass swagger_js_url/swagger_css_url/swagger_favicon_url, however it not give values when call it in applications.

can not access `cdn.jsdelivr.net` in China, I want to changed it to a local static file or other cdn.

https://github.com/fastapi/fastapi/pull/11574
https://fastapi.tiangolo.com/how-to/configure-swagger-ui/

### Operating System

Windows

### FastAPI Version

0.115

### Pydantic Version

2.9.2

### Python Version

3.11.2

**Comment by jgould22** [comment, marked as answer]
Are you overriding the url as described in the doc you linked?

It might look something like 

```python
app = FastAPI(swagger_ui_parameters={
"swagger_js_url": "some other url or path",
"swagger_css_url": "some other url or path"
 })
```
